packages/vite-plugin-cloudflare/playground/sensitive-files/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+	async fetch() {
+		return new Response("Worker response");
```

Comment:
Once #9510 lands we should also check that `env.ASSETS.fetch()` also doesn't expose these files.